source/connect/mongoclient.txt

```diff
@@ -64,94 +81,45 @@ A standard connection string includes the following pieces:
-
-Connect to Atlas
-~~~~~~~~~~~~~~~~
-
-To connect to a MongoDB deployment on Atlas, create a client. You can
-create a client that uses your connection string and other 
-client options by passing a ``MongoClientSettings`` object to the ``MongoClient``
-constructor.
+For more information about creating a connection string, see
+:manual:`Connection Strings </reference/connection-string?tck=docs_driver_csharp>` in the
+MongoDB Server documentation.
 
-To specify your connection URI, pass it to the ``FromConnectionString()``
-method, which returns a new ``MongoClientSettings`` instance. To specify any other
-client options, set the relevant fields of the ``MongoClientSettings`` object.
+MongoClient
+-----------
 
-You can set the {+stable-api+} version as a client option to avoid
-breaking changes when you upgrade to a new server version. To 
-learn more about the {+stable-api+} feature, see the :ref:`{+stable-api+} page
-<csharp-stable-api>`.
+To create a connection to MongoDB, pass a connection URI to the
+``MongoClient`` constructor. In the following example, the driver uses a sample
+connection URI to connect to a MongoDB deployment running on port ``27017`` of ``localhost``:abbreviation:
 
-The following code shows how you can specify the connection string and
-the {+stable-api+} client option when connecting to a MongoDB
-deployment and verify that the connection is successful:
+.. code-block:: csharp
 
-.. literalinclude:: /includes/fundamentals/code-examples/connection/AtlasConnection.cs
-   :language: csharp
-   :start-after: // start atlas connection
-   :end-before: // end atlas connection
+   const string connectionUri = "mongodb://localhost:27017/";
+   var client = new MongoClient(connectionUri);
 
-.. tip::
+Configuring the MongoClient
+---------------------------
 
-   Follow the :atlas:`Atlas driver connection guide </driver-connection?tck=docs_driver_nodejs>`
-   to retrieve your connection string.
+You can configure settings for the ``MongoClient`` object by passing a
+``MongoClientSettings`` object to the constructor. The following example creates a
+``MongoClient`` object and sets the ``UseTls`` property to ``true``:
 
-Connect to a Replica Set
-~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: csharp
 
-To connect to a replica set deployment, specify the hostnames (or IP addresses) and 
-port numbers of the members of the replica set.
+   var mongoClientSettings = MongoClientSettings
+       .FromConnectionString("mongodb://localhost:27017/");
+   mongoClientSettings.UseTls = true;
+   var client = new MongoClient(mongoClientSettings);
 
-If you aren't able to provide a full list of hosts in the replica set, you can 
-specify one or more of the hosts in the replica set and instruct the driver to 
-perform automatic discovery in one of the following ways:
+For a full list of the settings you can configure, see the
+`MongoClientSettings <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.MongoClientSettings.html>`_
+API documentation.
```

Comment:
S: link to on domain settings docs: https://deploy-preview-621--docs-csharp.netlify.app/connect/connection-options/